src/main/java/org/commonjava/util/partyline/impl/infinispan/model/InfinispanJFS.java

```diff
@@ -168,11 +185,25 @@ public void onCacheCreatedEvent( CacheEntryEvent event )
             if ( event.getKey() == this.key )
```

Comment:
We need to use Object.equals() instead of == here.

Reply:
We need to change the UUID fields to Strings, so we can escape this .equals() mess:

https://www.tutorialspoint.com/java/util/uuid_equals.htm